testing/src/main/java/org/oppia/android/testing/media/TestMediaPlayer.kt

```diff
@@ -0,0 +1,51 @@
+package org.oppia.android.testing.media
+
+import android.content.Context
+import android.media.MediaPlayer
+
+/**
+ * Helper class to test Media Player related test cases in both Robolectric & Espresso.
+ */
+interface TestMediaPlayer {
+
+  /**
+   * Add Media details and Audio Data Source
+   *
+   * Adding the required details of media which we want our test cases to run on.
+   * [context] the test environment context
+   * [explorationId1], [explorationId2] Id of the exploration
+   * [audioFileName1], [audioFileName2] name of the  audio file
+   * [mediaPlayer] Media Player object we use in our test case
```

Comment:
We should use block tags (https://developer.android.com/kotlin/style-guide#block_tags) when writing KDocs. In this case, ``@param``. Please refer to other KDocs present in the codebase.

Reply:
Done

Reply:
resolving on behalf of Ben